src/ch/ehi/sqlgen/generator_impl/jdbc/GeneratorGeoPackage.java

```diff
@@ -448,4 +418,21 @@ static public String getGpkgGeometryTypename(int type)
 				throw new IllegalArgumentException();
 		 }
 	}
+    @Override
+    public void visit1End() throws IOException {
+        String stmt=totalScript.toString();
+        try{
+            Statement dbstmt = conn.createStatement();
```

Comment:
wrap it in if(conn!=null). 

conn is null if --createscipt function is used in ili2db (see failing SimpleGpkgTest.createScriptFromIliCoord() in ili2db)

Reply:
Done!